src/shapes/Text.ts

```diff
@@ -303,11 +315,13 @@ export class Text extends Shape<TextConfig> {
           lineTranslateX += this.measureSize(letter).width + letterSpacing;
         }
       } else {
+        context.trySetLetterSpacing(letterSpacing);
```

Comment:
I suggest to set letter spacing similar to all other attributes. Like `context.setAttr('letterSpacing', value);` or direct set.